### PR TITLE
Run 32 bit linux test as github action

### PR DESCRIPTION
**Pull request by svenklemm**
Unfortunately some of the github actions do not work properly with the i386 image so we can't use caching here.

Run log: https://github.com/svenklemm/timescaledb/actions/runs/115996513

### CODE REVIEW

File: .github/workflows/alpine-32bit-build-and-test.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+    - name: Build pg_isolation_regress
+      run: |
+        wget -q -O postgresql.tar.bz2 https://ftp.postgresql.org/pub/source/v${{ matrix.pg }}/postgresql-${{ matrix.pg }}.tar.bz2
+        mkdir -p ~/postgresql
+        tar --extract --file postgresql.tar.bz2 --directory ~/postgresql --strip-components 1
+        cd ~/postgresql
+        ./configure --prefix=/usr/local --enable-debug --enable-cassert --with-openssl --without-readline --without-zlib
+        make -C src/test/isolation
+        chown -R postgres:postgres ~/postgresql
```
Comment (mkindahl) [inline .github/workflows/alpine-32bit-build-and-test.yaml:38]: You might want to consider if you should cache this: https://help.github.com/en/actions/configuring-and-managing-workflows/caching-dependencies-to-speed-up-workflows

Comment (mkindahl) [reply .github/workflows/alpine-32bit-build-and-test.yaml:38]: Sorry, didn't read your comment. :)

File: .github/workflows/alpine-32bit-build-and-test.yaml [hunk 2]
```diff
@@ -0,0 +1,73 @@
+name: Linux 32-bit x86
+on:
+  push:
+#    branches:
+#      - prerelease_test
```
Comment (k-rus) [inline .github/workflows/alpine-32bit-build-and-test.yaml:5]: Is there any specific reason to keep this commented? Is it correct that this workflow runs on any commit to any branch in the repo? Will it be better to do in the same way as @erimatnor did for Windows build, when workflows run on pull requests and push to prerelease branch?

**Comment by svenklemm** [comment]
This is supposed to replace the current i386 tests so it will run every day on master branch and on push to specific branches (prerelease_test, maybe others), its not part of the PR yet cause I want the workflow to run on my PR branch to make sure it works.

Regarding caching while we could built specific 32bit docker images with the tools I decided not to do it for this PR because this workflow is not run that often.

**Pull request merged**